components/ADempiere/Form/VAllocation/components/Invoce.vue

```diff
@@ -0,0 +1,268 @@
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <https:www.gnu.org/licenses/>.
+-->
+
+<template>
+  <div style="display: contents;height: 100% !important;">
+    <el-table
+      :data="tableData"
+      border
+      style="width: 100%"
+    >
+      <el-table-column
+        v-for="(header, key) in headers"
+        :key="key"
+        prop="id"
+        :type="header.type"
+        :label="header.label"
+      />
+    </el-table>
+    <el-card class="box-card">
+      <div slot="header" class="clearfix" style="text-align: center;">
+        <b>
+          {{ 'Detalles de la Factura' }}
+        </b>
+      </div>
+      <div id="description-payment" class="description-payment" style="display: flex;">
+        <el-card
+          class="box-card"
+          style="width: 50%;float: left;"
+        >
+          <div>
+            <el-form
+              :inline="true"
+              label-position="left"
+              style="padding: 10px !important;"
+            >
+              <el-col :span="24">
+                <el-form-item
+                  label="Cantidad de Registro:"
+                >
+                  <b>
+                    5
+                  </b>
+                </el-form-item>
+              </el-col>
+              <el-col :span="24">
+                <el-form-item
+                  label="Total Factura:"
+                >
+                  <b>
+                    8.506,23
+                  </b>
+                </el-form-item>
+              </el-col>
+              <el-col :span="24">
+                <el-form-item
+                  label="Convertido:"
+                >
+                  <b>
+                    530,63
+                  </b>
+                </el-form-item>
+              </el-col>
+            </el-form>
+          </div>
+        </el-card>
+        <el-card
+          class="box-card"
+          style="width: 50%;float: right;"
+        >
+          <div>
+            <el-form
+              :inline="true"
+              label-position="left"
+              style="padding: 10px !important;"
+            >
+              <el-col :span="24">
+                <el-form-item
+                  label="Pagado:"
+                >
+                  <b>
+                    350,63
+                  </b>
+                </el-form-item>
+              </el-col>
+              <el-col :span="24">
+                <el-form-item
+                  label="Asignado:"
+                >
+                  <b>
+                    170,00
+                  </b>
+                </el-form-item>
+              </el-col>
+              <el-col :span="24">
+                <el-form-item
+                  label="Monto Abierto:"
```

Comment:
@elsiosanchez please add translation.

Reply:
Added the translation in the following commit [68211fe](https://github.com/solop-develop/frontend-default-theme/pull/442/commits/68211fe355873ffe49725eb888013312491e1af4)